juju/juju
# Add annotations to the cache for objects that have annotations.

PR #11244 added annotations to the multiwatcher types. This PR threads those annotations through to the cache.

Additionally we add model annotations to the model summaries so model annotations can be used to mute models for errors in the dashboard.

Facade schema is updated, and a current PR on pylibjuju will be updated to use it once this is landed.

## QA steps

```sh
cd core/cache
go test -check.v
cd ../../workers/modelcache
go test -check.v
```
